labs/firepower-restapi-103/1.md

```diff
@@ -1,69 +1,70 @@
 # FMC - REST - API - 103
 
-## Lab Objective
+## Lab objective
```

Comment:
I think I've mostly seen "Objectives" or "Objective" for this heading - adding Lab is probably not consistent.